kafka/consumer-group/consumer_group.go

```diff
@@ -340,7 +352,15 @@ func (cg *ConsumerGroup) topicConsumer(topic string, messages chan<- *sarama.Con
 	// Consume all the assigned partitions
 	var wg sync.WaitGroup
 	for _, pid := range myPartitions {
-
+		//Create PartitionInfo and send it for ingestion through the partition channel
+		//In case of re-balancing this function will be triggered again and the latest information will be sent
+
+		metricName := name + "." + topic + "." + cg.instance.ID + "." + time.Now().Format(time.RFC850)
```

Comment:
what is instance id here?

Reply:
It is the consumer group ID to track which partition belongs to which consumer group

Reply:
if we look at consumer group-id, it's built using hostname and a unique id. can you please check if this uniqueness is something we can capture using host filter from which metric is emitted. if that's present then we don't need it.